runelite-client/src/main/resources/net/runelite/client/plugins/microbot/shortestpath/spirit_trees.tsv

```diff
@@ -149,130 +149,130 @@
-2556 3259 0	2542 3170 0	Travel Spirit tree 26263				2	1: Tree Gnome Village
-2557 3260 0	2542 3170 0	Travel Spirit tree 26263				2	1: Tree Gnome Village
-2557 3261 0	2542 3170 0	Travel Spirit tree 26263				2	1: Tree Gnome Village
-2556 3262 0	2542 3170 0	Travel Spirit tree 26263				2	1: Tree Gnome Village
-2555 3262 0	2542 3170 0	Travel Spirit tree 26263				2	1: Tree Gnome Village
-2554 3261 0	2542 3170 0	Travel Spirit tree 26263				2	1: Tree Gnome Village
-2554 3260 0	2542 3170 0	Travel Spirit tree 26263				2	1: Tree Gnome Village
-2555 3259 0	2461 3444 0	Travel Spirit tree 26263				2	2: Gnome Stronghold
-2556 3259 0	2461 3444 0	Travel Spirit tree 26263				2	2: Gnome Stronghold
-2557 3260 0	2461 3444 0	Travel Spirit tree 26263				2	2: Gnome Stronghold
-2557 3261 0	2461 3444 0	Travel Spirit tree 26263				2	2: Gnome Stronghold
-2556 3262 0	2461 3444 0	Travel Spirit tree 26263				2	2: Gnome Stronghold
-2555 3262 0	2461 3444 0	Travel Spirit tree 26263				2	2: Gnome Stronghold
-2554 3261 0	2461 3444 0	Travel Spirit tree 26263				2	2: Gnome Stronghold
-2554 3260 0	2461 3444 0	Travel Spirit tree 26263				2	2: Gnome Stronghold
-2555 3259 0	3185 3508 0	Travel Spirit tree 26263				2	4: Grand Exchange
-2556 3259 0	3185 3508 0	Travel Spirit tree 26263				2	4: Grand Exchange
-2557 3260 0	3185 3508 0	Travel Spirit tree 26263				2	4: Grand Exchange
-2557 3261 0	3185 3508 0	Travel Spirit tree 26263				2	4: Grand Exchange
-2556 3262 0	3185 3508 0	Travel Spirit tree 26263				2	4: Grand Exchange
-2555 3262 0	3185 3508 0	Travel Spirit tree 26263				2	4: Grand Exchange
-2554 3261 0	3185 3508 0	Travel Spirit tree 26263				2	4: Grand Exchange
-2554 3260 0	3185 3508 0	Travel Spirit tree 26263				2	4: Grand Exchange
-2555 3259 0	2488 2850 0	Travel Spirit tree 26263				2	5: Feldip Hills
-2556 3259 0	2488 2850 0	Travel Spirit tree 26263				2	5: Feldip Hills
-2557 3260 0	2488 2850 0	Travel Spirit tree 26263				2	5: Feldip Hills
-2557 3261 0	2488 2850 0	Travel Spirit tree 26263				2	5: Feldip Hills
-2556 3262 0	2488 2850 0	Travel Spirit tree 26263				2	5: Feldip Hills
-2555 3262 0	2488 2850 0	Travel Spirit tree 26263				2	5: Feldip Hills
-2554 3261 0	2488 2850 0	Travel Spirit tree 26263				2	5: Feldip Hills
-2554 3260 0	2488 2850 0	Travel Spirit tree 26263				2	5: Feldip Hills
-2555 3259 0	3274 6123 0	Travel Spirit tree 26263			Song of the Elves	2	6: Prifddinas
-2556 3259 0	3274 6123 0	Travel Spirit tree 26263			Song of the Elves	2	6: Prifddinas
-2557 3260 0	3274 6123 0	Travel Spirit tree 26263			Song of the Elves	2	6: Prifddinas
-2557 3261 0	3274 6123 0	Travel Spirit tree 26263			Song of the Elves	2	6: Prifddinas
-2556 3262 0	3274 6123 0	Travel Spirit tree 26263			Song of the Elves	2	6: Prifddinas
-2555 3262 0	3274 6123 0	Travel Spirit tree 26263			Song of the Elves	2	6: Prifddinas
-2554 3261 0	3274 6123 0	Travel Spirit tree 26263			Song of the Elves	2	6: Prifddinas
-2554 3260 0	3274 6123 0	Travel Spirit tree 26263			Song of the Elves	2	6: Prifddinas
```

Comment:
Why have these ids been changed? Were they incorrect?

Reply:
Yeah, I couldn't get this provided ID to correlate to anything, game object ID in-game is different!!